docs/market_analysis.rst

```diff
@@ -0,0 +1,277 @@
+
+When this module is enabled it will start recording the lending rates for the market in an sqlite database. This will be seen in the market_data folder for your bot. This supersedes the previous method of storing it in a file. The files can be removed if you have them from older versions of the bot.
+
+There will be a DB created for each currency you wish to record. These can be enabled in the `analyseCurrencies`_ configuration option.  
+
+ .. warning:: The more currencies you record, the more data stored on disk and CPU processing time will be used. You will also not get as frequent results for the currencies, i.e. You may have trouble getting results for your configured ``analyseUpdateInterval`` This is explained further in the `Recording currencies`_ section. 
+
+A quick list of each config option and what they do
+
+========================= =============================================================================================
+`analyseCurrencies`_      A list of each currency you wish to record and analyse
+`analyseUpdateInterval`_  The frequency between rates requested and stored in the DB
+`lendingStyle`_           The percentage used for the percentile calculation
+`percentile_seconds`_     The number of seconds to analyse when working out the percentile
+`MACD_long_win_seconds`_  The number of seconds to used for the long moving average
+`MACD_short_win_seconds`_ The number of seconds to used for the short moving average
+`keep_history_seconds`_   The age (in seconds) of the oldest data you wish to keep in the DB
+`recorded_levels`_        The depth of the lending book to record in the DB, i.e. how many unfilled loans
+`data_tolerance`_         The percentage of data that can be ignore as missing for the time requested in
+                          ``percentile_seconds`` and ``MACD_long_win_seconds``
+`daily_min_method`_       Which method (MACD or percentile) to use for the daily min calculation
+`MACD_multiplier`_        Only valid for MACD method. The figure to scale up the returned rate value from the MACD calculation
+========================= =============================================================================================
+
+The module has two main methods to calculate the minimum rate:
+
+Percentile
+``````````
+This method takes all the data for the given time period (`percentile_seconds`_) and works out the Xth percentile (`lendingStyle`_) for that set of data. For example if you are using a ``lendingStyle`` of 85 and you had a list of rates like so
+
+  :Example: 0.04, 0.04, 0.05, 0.05, 0.05, 0.05, 0.06, 0.06, 0.06, 0.07, 0.07, 0.07, 0.08, 0.08, 0.09, 0.09, 0.09, 0.10, 0.10, 0.10
+
+The 85th percentile would be 0.985 because 85% of rates are below this. The following configuration options should be considered when using the percentile calculation method:-
```

Comment:
So I'm not sure what either of you are getting at here. 

This was just an example of how the percentile calculation works (for those who don't know). I haven't gone into the details of which DB etc it uses. This is still the old analysis method we had. It just uses the DB for the data now.